src/components/landing/LandingPage.jsx

```diff
@@ -20,12 +20,6 @@ export default function Landing() {
         Software tools for automating 3D printability checks of 3D model files
       </h2>
 
-      <img
-        id="coming-soon-image"
-        src="assets/coming-soon.png"
-        alt="Coming Soon"
-      />
-
```

Comment:
When this gets removed it puts the two other things that were above and below it next to each other rather than on top of each other. Can we format it back to on top of each other?
![next-to-each-ther](https://github.com/rknizzle/propermesh-view/assets/40612461/5b5a1073-8642-494b-be9e-9c24ebbccfe9)